app/src/main/res/values/strings.xml

```diff
@@ -45,7 +45,7 @@
     <string name="invalid_url">Invalid URL</string>
     <string name="subject">Topic</string>
     <string name="reply_to_sender">Reply to sender</string>
-    <string name="google_app_login_failed">Google apps login failed for unknown reasons, please contact your distributor.</string>
+    <string name="google_app_login_failed">Google Apps login failed for unknown reasons, please contact your distributor.</string>
```

Comment:
Hey, @kunall17! Thank you for your feeback :)

"Apps" in this case is a proper noun, since it's the name of the product offered by Google (check [this](https://developers.google.com/google-apps/) out). Hence the capitalization.